cogs/tags.py

```diff
@@ -1,54 +1,79 @@
-from discord.ext import commands
-import discord
-
 import asyncio
+from datetime import datetime
+
 import asyncpg
+import discord
+from config import TAGS_LOG_CHANNEL_ID
+from discord.ext import commands
 
+from .utils.checks import is_admin, is_engineer_check, is_staff
 from .utils.DataBase.tag import Tag
-from .utils.checks import is_engineer_check, is_admin, is_staff
+
+EMOJIS = [
+    "\N{WHITE HEAVY CHECK MARK}",
+    "\N{CROSS MARK}",
+]
 
 
 def setup(bot):
     bot.add_cog(TagCommands(bot=bot))
 
 
 class TagCommands(commands.Cog, name="Tags"):
-    def __init__(self, bot):
+    def __init__(self, bot: commands.Bot):
         self.bot = bot
+        self.log_channel = self.bot.get_channel(TAGS_LOG_CHANNEL_ID)
 
     def cog_check(self, ctx):
         if ctx.guild is None:
             return False
+
         return True
 
+    ####################################################################################################################
+    # Commands
+    ####################################################################################################################
+
     @commands.group(invoke_without_command=True)
     async def tag(self, ctx, *, name: lambda inp: inp.lower()):
         """Main tag group."""
+        name = await commands.clean_content().convert(ctx=ctx, argument=name)
         tag = await self.bot.db.get_tag(guild_id=ctx.guild.id, name=name)
 
         if tag is None:
             await ctx.message.delete(delay=10.0)
-            message = await ctx.send('Could not find a tag with that name.')
+            message = await ctx.send("Could not find a tag with that name.")
             return await message.delete(delay=10.0)
 
         await ctx.send("{}".format(tag.text))
-        await self.bot.db.execute("UPDATE tags SET uses = uses + 1 WHERE guild_id = $1 AND name = $2",
-                                  ctx.guild.id, name)
+        await self.bot.db.execute(
+            "UPDATE tags SET uses = uses + 1 WHERE guild_id = $1 AND name = $2",
+            ctx.guild.id,
+            name,
+        )
 
     @tag.command()
     async def info(self, ctx, *, name: lambda inp: inp.lower()):
         """Get information regarding the specified tag."""
+        name = await commands.clean_content().convert(ctx=ctx, argument=name)
```

Comment:
I think `name: commands.clean_content` then `name = name.lower()` would be clearer than invoking the converter for clean content.